src/platform/plugins/shared/dashboard/public/dashboard_api/get_serialized_state.ts

```diff
@@ -138,6 +138,8 @@ export const getSerializedState = ({
     { embeddablePersistableStateService: embeddableService }
   );
 
+  // TODO Provide tags as an array of tag names in the attribute. In that case, tag references
```

Comment:
Do we have a followup issue for this?

Reply:
https://github.com/elastic/kibana/issues/216346